ads/_a4a-config.js

```diff
@@ -57,3 +57,12 @@ export const a4aRegistry = {
 if (getMode().localDev || getMode().test) {
   a4aRegistry['fake'] = fakeIsA4AEnabled;
 }
+
+/**
+ * An object mapping signing server names to their corresponding URLs.
+ * @type {!Object<string, string>}
+ */
+export const signingServerURLs = {
```

Comment:
To clarify, each signing service (not each ad network) gets its own URL. Each ad network indicates which signing services it uses, and we fetch only those keys.